lib/webapp.js

```diff
@@ -530,9 +530,14 @@ function WebApp(options){
   if (options.sessionTimeoutMs) {
     sessionTimeoutMs = options.sessionTimeoutMs;
   }
+  let port = options.httpsPort;
+  if (!options.httpsPort) {
+    port = options.httpPort;
+  }
```

Comment:
Better way to do this is:
`const port = options.httpsPort ? options.httpsPort : options.httpPort;`
Allows use of const for read-only, and also does not need reassignment.